commands/log_writer_test.go

```diff
@@ -0,0 +1,53 @@
+package commands_test
+
+import (
```

Comment:
yeah, I wasn't sure I liked this either but it was so small that I wanted to leave it in commands until it really needed to be moved out. I could go either way on this and didn't love it.